dotty-tensorflow-examples/src/main/scala/me/shadaj/scalapy/tensorflow/example/BidirectionalLSTMExample.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from tensorflow import keras
+from tensorflow.keras import layers
+import time
+
+
+start_time = time.time()
+
+max_features = 200  # Only consider the top 20k words
+maxlen = 10  # Only consider the first 200 words of each movie review
+
+# Input for variable-length sequences of integers
+inputs = keras.Input(shape=(None,), dtype="int32")
+# Embed each integer in a 128-dimensional vector
+x = layers.Embedding(max_features, 128)(inputs)
+# Add 2 bidirectional LSTMs
+x = layers.Bidirectional(layers.LSTM(64, return_sequences=True))(x)
+x = layers.Bidirectional(layers.LSTM(64))(x)
+# Add a classifier
+outputs = layers.Dense(1, activation="sigmoid")(x)
+model = keras.Model(inputs, outputs)
+(x_train, y_train), (x_val, y_val) = keras.datasets.imdb.load_data(
+    num_words=max_features
+)
+print(len(x_train), "Training sequences")
+print(len(x_val), "Validation sequences")
+x_train = keras.preprocessing.sequence.pad_sequences(x_train, maxlen=maxlen)
+x_val = keras.preprocessing.sequence.pad_sequences(x_val, maxlen=maxlen)
+model.compile("adam", "binary_crossentropy", metrics=["accuracy"])
+model.fit(x_train, y_train, batch_size=32, epochs=2, validation_data=(x_val, y_val))
+
+score = model.evaluate(x = x_val, y = y_val, verbose = 0)
+
+print("Test loss: " , score[0])
+print("Test accuracy: ", score[1])
+
+print("Total time: %s s" % (time.time() - start_time))
```

Comment:
Pls add the sample results to README in a separate section so that we don't forget about them